HousingSearchApi/V1/Boundary/Requests/GetAllTenureListRequest.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace HousingSearchApi.V1.Boundary.Requests
+{
+    public class GetAllTenureListRequest : GetTenureListRequest
+    {
+        [FromQuery(Name = "lastHitId")]
+        public string LastHitId { get; set; }
+
+        //in order to use sorting together with last hit id, this must be provided as well 
+        [FromQuery(Name = "lastHitTenureStartDate")]
+        public string LastHitTenureStartDate { get; set; }
```

Comment:
Yes, `LastHitId` is the tenure id. The way this is designed to work is that when you use the `LastHitTenureStartDate` together with `LastHitId` style paging/load more feature the ES client provides the values for the next call. It always gives you the next two required values. These are available in the `Sort` array in the `Hit` object in the ES client response. So all the client has to do is to grab the values from the response and pass them back in the next call, so there's no need to calculate or fecth the start date at any point either on the API or the client. This offloads the responsibility fully to the ES client and since it's in charge of the values it wants that ensures things work as expected.